src/modules/auth/components/email-verification/check-mail-box.tsx

```diff
@@ -0,0 +1,27 @@
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
+import EmailOutlinedIcon from '@mui/icons-material/EmailOutlined';
+import { checkMailBoxStyles } from './check-mail-box.styles';
+
+export function CheckMailbox({ email }: { email: string }) {
+  return (
+    <Container maxWidth="xs" sx={checkMailBoxStyles.container}>
+      <Paper elevation={3} sx={checkMailBoxStyles.paper}>
+        <EmailOutlinedIcon sx={checkMailBoxStyles.icon} />
+        <Typography variant="h5" gutterBottom={true}>
+          Verify Your Email
```

Comment:
add translation for static texts

Reply:
where can I add it?

Reply:
public json file for English and Russian and translation keys constant for referencing json.

 you can use useTranslation hook. 

for reference you can see how I did it in auth or in any other pages.

First write English one then you can use chatgpt for generating Russian json based on English json